--- init.php.backup	2024-02-20 19:06:59.573563597 +0900
+++ init.php	2024-02-20 19:07:13.397635945 +0900
@@ -154,7 +154,7 @@
 						. 'AllowOverride None in your configuration may be blocking the use of .htaccess or %s is not resolvable.'),
 						$fm_name, $_SERVER['SERVER_NAME']);
 				if ($single_check) {
-					bailOut($message);
+					// bailOut($message);
 				} else {
 					$requirement_check .= displayProgress(_('Test Rewrites'), false, 'display', $message);
 					$error = true;
